src/views/Index.svelte

```diff
@@ -889,8 +890,9 @@
                      2.
                   </span>
                   <div class="tm-feature-content">
-                     <h5>Era Swap Ecosystem (ESE)</h5>
-                     <p>includes all inter-linked platforms where ES can be utilized for multiple utilities. The ecosystem consists of platforms offering P2P services and usage to community members.</p>
+                     <h5>Era Swap Token (ES)</h5>
+                     <p>ES is a native token of Era Swap Blockchain Network proof-of-stake Consensus Protocol, designed to optimize utilization in Era Swap Ecosystem & Strengthen ESN for long-term viability. It allows P2P Exchange using Smart Contracts within Era Swap Decentralized Economy.
+</p>
```

Comment:
The paragraph tag is closed in new line. I think we should start to enforce prettier style checks.